pkg/server/tasks.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019, Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"time"
+)
+
+const (
+	// This command refers to the shell script in the dev-scripts repository.
+	deployBoostrapVMCommand = "./05_deploy_bootstrap_vm.sh"
+)
+
+// This is an example of a task that might take a while to finish.  It takes a
+// Notification channel as an argument.  You can use this channel to send
+// Notification messages to the browser.
+func performLongTask(notificationChannel chan Notification) {
+	n := NewNotification("Started a long running task", RUNNING)
+	notificationChannel <- n
+
+	// Do important work for 10 seconds
+	time.Sleep(10 * time.Second)
+
+	n2 := NewNotification("Finished a long running task", SUCCESS)
+	notificationChannel <- n2
+}
+
+func createBootstrapVM(notificationChannel chan Notification) {
+	log.Print("Booting bootstrap VM")
+
+	n := NewNotification("Booting bootstrap VM", RUNNING)
+	notificationChannel <- n
+
+	cmd := exec.Command(deployBoostrapVMCommand)
```

Comment:
Shelling out is fine for now, but we can probably look at the openshift-install libvirt backend for a go-native way to do this